wpiutil/src/main/java/edu/wpi/first/util/TimestampSource.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.util;
+
+/**
+ * Options for where the timestamp an {@link RawFrame} was captured at can be measured relative to.
+ */
+public enum TimestampSource {
+  /** unknown. */
+  kUnknown(0),
+  /**
+   * wpi::Now when the new frame was dequeued by CSCore. Does not account for camera exposure time
+   * or V4L latency.
+   */
+  kFrameDequeue(1),
+  /** End of Frame. Same as V4L2_BUF_FLAG_TSTAMP_SRC_EOF, translated into wpi::Now's timebase. */
+  kV4lEoF(2),
```

Comment:
The capitalization here is terrible... either the `f` needs to be lowercase, or it should be all caps.

Reply:
So kV4lEof or kV4lEOF? Happy with either - I had o lowercase since that's how I'd write it out

Reply:
The capitals would be on the first letter of words, so it would be `kV4LEOF` or `kV4lEof`.

Reply:
How's the new spelling in 5da9d3cfb16edcc67ba6f19632fd82de5e1cc0c9 ?